std/std.zig

```diff
@@ -121,6 +121,16 @@ pub struct OutStream {
             }
         }
     }
+
+    pub fn close(os: &OutStream) -> %void {
+        const closed = close(os.fd);
+        if (closed < 0) {
+            return switch (-closed) {
```

Comment:
According to the man pages on my system, close can return these errors:

```
ERRORS
       EBADF  fd isn't a valid open file descriptor.

       EINTR  The close() call was interrupted by a signal; see signal(7).

       EIO    An I/O error occurred.
```

EBADF should return `error.BadFd`.

EINTR should return `error.SigInterrupt`.

the EIO and else cases look good.